cloudwatch/helpers.go

```diff
@@ -115,3 +116,13 @@ func parseDataMapTags(e *Event, logTags []string, template string) (string, erro
 		}
 	})
 }
+
+// truncateEvent reduces an Event to the Cloudwatch maximum of 256KB.
+// This function also returns the event byte count.
+func truncateEvent(event []byte) (string, int) {
+	if len(event) <= maximumBytesPerEvent {
+		return string(event), len(event)
+	}
+
```

Comment:
To test this, I added a line here like this:
```go
	logrus.Errorf("Truncated %d bytes from an event.", len(event)-maximumBytesPerEvent)
```

And then sent a large log like this:
```bash
cat /dev/urandom | tr -dc 'a-zA-Z0-9' | head --bytes 300000 | logger
```
**But that didn't work.** rsyslog or the syslog input breaks the message up before it gets to this plugin. I'm not real sure how to test this properly in my environment. I knows nothing is broken, but I'm not able to easily validate that this truncates a message that's too large.

Thoughts on adding the log line here permanently?